embedding/embedlite/embedshared/EmbedLiteViewBaseChild.cpp

```diff
@@ -115,6 +115,8 @@ EmbedLiteViewBaseChild::EmbedLiteViewBaseChild(const uint32_t& aWindowId, const
   , mVirtualKeyboardHeight(0)
   , mIMEComposing(false)
   , mPendingTouchPreventedBlockId(0)
+  , initialized(false)
+  , destroyAfterInit(false)
```

Comment:
```suggestion
  , mDestroyAfterInit(false)
```

Same for mInitialized